uktrade/data-hub-frontend
# Remove custom details polyfill

## Description of change

`<details>` and `<summary>` is not supported in IE. As a result, usage of the `<details>` element across the FE is not functioning correctly for IE users.

Currently there is a `~/assets/javascripts/vendor/details.polyfill.js` which does not appear to be functioning correctly. As we have a dependency on `govuk-frontend` there is an implementation of a details polyfill so we can remove our custom implementation.

## Test instructions

Browse to any `<details>` (collapsible block) in IE and after this change it should now be functioning correctly.
 
## Screenshots
### Before

![Screenshot 2019-08-08 at 20 27 20](https://user-images.githubusercontent.com/1150417/62731919-1b838c00-ba1b-11e9-91ae-283743219fa6.png)

### After 

![Screenshot 2019-08-08 at 20 27 26](https://user-images.githubusercontent.com/1150417/62731929-20e0d680-ba1b-11e9-8f01-07bdda8cd42f.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
